test/fixtures/data-driven-tests.json

```diff
@@ -2928,794 +2928,6 @@
             "type": "Amount",
             "error": "10000000000000000000 absolute XRP is bigger than max native value 100000000000.0",
             "is_negative": true
-        },
```

Comment:
As far as I can tell, all of these fixtures are never tested. They are also rather silly tests IMO because all they do is test what is already defined in `definitions.json`. We should be testing using more full examples. Will be adding tests later when this solidifies, as some of this is subject to change.